### PR TITLE
Use the new puppet GPG key

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

File: manifests/repo/yum.pp [hunk 1]
```diff
@@ -22,7 +22,7 @@
         baseurl  => "http://yum.puppetlabs.com/el/${::operatingsystemmajrelease}/${::puppet::collection}/${::architecture}",
         enabled  => '1',
         gpgcheck => '1',
-        gpgkey   => 'https://yum.puppetlabs.com/RPM-GPG-KEY-puppetlabs',
+        gpgkey   => 'https://yum.puppetlabs.com/RPM-GPG-KEY-puppet',
```
Comment (rendhalver) [inline manifests/repo/yum.pp]: Can you put these in a separate PR please.

Comment (hrpatel) [inline manifests/repo/yum.pp]: Doh.. I thought GitHub would be smart enough to ignore the other commit.. my bad. 

I'll sync my fork and send another one..

**Comment by hrpatel** [comment]
Closing to make a new one without the extra commit
